src/components/Escrow/Escrow.tsx

```diff
@@ -28,7 +28,7 @@ export const EscrowContainer: React.FC<IEscrowContainer> = ({
   const { rpcUrl } = networkMap[network];
 
   const { data: queryResponse } = useQuery(ESCROW_STATS);
-  if (queryResponse) {
+  if (queryResponse?.escrowStatistics) {
```

Comment:
@foufrix this change checks if escrowStatistics returns `undefined`. This happens to subgraphs where HMTokens were deployed by Multichain (Moonbeam, Optimism, and probably BSC too).

This is a temporary fix, but the optimal solution would be fixing the entities in the subgraph folder at [hmt-escrow](https://github.com/humanprotocol/hmt-escrow) such as what has been done here: https://github.com/humanprotocol/humansubgraph/tree/rinkeby

Reply:
Works for me, as we only use `queryResponse.escrowStatistics` in the function, it make sense to check for its existence